src/win/stream.c

```diff
@@ -48,6 +48,48 @@ int uv_listen(uv_stream_t* stream, int backlog, uv_connection_cb cb) {
 }
 
 
+int uv_reject(uv_stream_t* client) {
+  int err = 0;
+  uv_pipe_t* pipe_client;
+  uv_tcp_accept_t* req;
+
+  if (client == NULL) {
+    return uv_translate_sys_error(ERROR_INVALID_PARAMETER);
+  }
+
+  switch (client->type) {
+    case UV_NAMED_PIPE:
+      pipe_client = (uv_pipe_t*)client;
+      if (pipe_client->u.fd == -1) {
+        CloseHandle(pipe_client->handle);
+      } else {
+        _close(pipe_client->u.fd);
```

Comment:
Hum, aren't we actually closing the pipe server here?! This is not the accepted handle, it's the server itself.